### PR TITLE
Migrating to jsPsych 7.2.1

### PR DESCRIPTION
**Pull request by Kostimbus**
- All experiments besides 'color-toj-negation-06.js' are migrated to version 7.2.1.
- Utility functions were also updated according to the requirements
- 'package.json' now contains relevant versions of dependencies
- Minor update in 'main.scss'

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,32 +1,40 @@
...
   },
   "devDependencies": {
-    "jspsych-builder": "2.1.0"
+    "jspsych-builder": "^4.2.1"
   },
-  "browserslist": [
-    "last 2 versions",
-    "ie >= 9",
-    "ios >= 7"
-  ],
```
Comment (m-neuss) [inline package.json:28]: Just making sure: Is it intended to remove browserslist?

File: src/util/introduction.js [hunk 2]
```diff
@@ -96,94 +95,50 @@ export function addIntroduction(timeline, options) {
...
-    type: "html-button-response",
-    stimulus: () => {
-      return `<iframe class="technical-instruction" src="media/misc/technical_instructions_sound_${globalProps.instructionLanguage}.html"></iframe>`;
-    },
-    choices: () =>
-      globalProps.instructionLanguage === "en"
-        ? ["Computer sounds are enabled"]
-        : ["Der Ton ist eingeschaltet"],
-  });
-
```
Comment (m-neuss) [inline src/util/introduction.js]: Why are these instructions being removed? Shouldn't they be migrated to jsPsych 7.2.1 instead?

**Review by m-neuss — COMMENTED** [review]
Made a test run of every experiment (except color-toj-negation-06.js) with jsPsych 7.2.1 and everything seems to work technically correct.

Nevertheless, I have some further questions/suggestions:

- To be consistent with changes in other experiments, perhaps the following should be considered:
  - bump version number of:
    - color-toj-negation-03b.js
    - color-toj-negation-05.js
    - quick-toj.js
  - rename constant `cond` to `condition` in:
    - color-toj.js
    - color-toj-hue.js

File: src/util/introduction.js (hunk 2 above)
Comment (Kostimbus) [reply src/util/introduction.js]: Actually, I don't how I've missed this.

P.S Fixed

**Comment by Kostimbus** [comment]
Have noticed the same thing with cond variable in quick-toj and some other little issues. Idea with bumping version has the meaning - done!

**Review by m-neuss — COMMENTED** [review]
Tested all experiments again. Everything seems to work as intended.

- All instructions from introduction.js are now implemented/migrated correctly.
- Version numbers are consistently bumped.
- `condition` is consistently renamed

File: package.json (hunk 1 above)
Comment (chisburger) [reply package.json:28]: It was changed upstream by the original author; thus, this change should be okay.

**Comment by caiusno1** [comment]
@chisburger has rebased this branch to ncbanh\jsPsych-update-to-v7-rebase. Therefore, I close this PR and open a new one for ncbanh\jsPsych-update-to-v7-rebase

**Pull request closed without merging**